requests_toolbelt/_compat.py

```diff
@@ -18,11 +18,13 @@
     from requests.packages.urllib3 import fields
     from requests.packages.urllib3 import filepost
     from requests.packages.urllib3 import poolmanager
+    from requests.packages.urllib3.util import timeout
 except ImportError:
     from urllib3 import connection
     from urllib3 import fields
     from urllib3 import filepost
     from urllib3 import poolmanager
+    from urllib3.util import timeout
 
 if requests.__build__ < 0x020800:
     gaecontrib = None
```

Comment:
Ah we already have something like that here. We also need to update the version of urllib3 here because we're only going to properly support this for versions `>= 0x021000`